src/app/(auth)/sign-up/SignUpForm.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+import { Input } from "@utilities/Input";
+import { SignUpValidationSchema } from "@auth/sign-up/signUpValidationSchema";
+import { Button } from "@utilities/Button";
+import { Form, Formik } from "formik";
+import { ZodError, z } from "zod";
+
+type SignUpFormValues = z.infer<typeof SignUpValidationSchema>;
+
+export const SignUpForm = () => {
+  
+  const validateForm = (values: SignUpFormValues) => {
+    try {
+      SignUpValidationSchema.parse(values);
+    } catch (error) {
+      if (error instanceof ZodError) {
+        return error.formErrors.fieldErrors;
+      }
+    }
+  };
+
+  const handleSubmit = async (values: SignUpFormValues) => {
+    try{
+      console.log(values);
+    } catch (error) {
+      console.error('error', error)
```

Comment:
Try to use as less console.log as you can. Errors should be displayed in the view. If something is in progress like the values above add comment with `TODO:` and some info about it